hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/TestNNThroughputBenchmark.java

```diff
@@ -166,4 +166,25 @@ public void testNNThroughputForAppendOp() throws Exception {
       }
     }
   }
+
+  /**
+   * This test runs {@link NNThroughputBenchmark} against a mini DFS cluster
+   * for block report operation.
+   */
+  @Test(timeout = 120000)
+  public void testNNThroughputForBlockReportOp() throws Exception {
+    final Configuration conf = new HdfsConfiguration();
+    conf.setInt(DFSConfigKeys.DFS_NAMENODE_MIN_BLOCK_SIZE_KEY, 16);
+    conf.setInt(DFSConfigKeys.DFS_BLOCK_SIZE_KEY, 16);
+    try (MiniDFSCluster cluster = new MiniDFSCluster.Builder(conf).
+        numDataNodes(3).build()) {
+      cluster.waitActive();
+      final Configuration benchConf = new HdfsConfiguration();
+      benchConf.setInt(DFSConfigKeys.DFS_NAMENODE_MIN_BLOCK_SIZE_KEY, 16);
+      benchConf.setInt(DFSConfigKeys.DFS_BLOCK_SIZE_KEY, 16);
+      NNThroughputBenchmark.runBenchmark(benchConf,
+          new String[]{"-fs", cluster.getURI().toString(), "-op",
+          "blockReport", "-datanodes", "3", "-reports", "2"});
```

Comment:
Would you fix the checkstyle warning in L187? Sorry my example was wrong.

Reply:
OK. I will fix some checkstyle formats, the updated format is as follows:
`
NNThroughputBenchmark.runBenchmark(benchConf,
          new String[]{"-fs", cluster.getURI().toString(), "-op",
              "blockReport", "-datanodes", "3", "-reports", "2"});
`

Reply:
Sorry, the information shown here cannot fully demonstrate my thoughts.
I will submit some updates.